### PR TITLE
Payments API

### PR DESCRIPTION
**Pull request by MrYadro**
Hello, take a look at this Payments API implementation, anything else needs to be fixed other than commenting everything new?

### CODE REVIEW

File: configs.go [hunk 1]
```diff
@@ -899,3 +899,87 @@ type ChatConfigWithUser struct {
...
+	return v, nil
+}
+
+func (config InvoiceConfig) method() string {
+	return "sendInvoice"
+}
+
+type ShippingConfig struct {
+	ShippingQueryID string // required
+	Ok              bool   // required
```
Comment (Syfaro) [inline configs.go]: Should be `OK` instead of `Ok`

File: configs.go [hunk 2]
```diff
@@ -899,3 +899,87 @@ type ChatConfigWithUser struct {
...
+	ShippingOptions *[]ShippingOption
+	ErrorMessage    string
+}
+
+type PreCheckoutConfig struct {
+	PreCheckoutQueryID string // required
+	Ok                 bool   // required
```
Comment (Syfaro) [inline configs.go]: Same capitalization here

File: helpers.go [hunk 3]
```diff
@@ -640,3 +640,24 @@ func NewCallbackWithAlert(id, text string) CallbackConfig {
 		ShowAlert:       true,
 	}
 }
+
+func NewInvoice(chatID int64, title, description, payload, providerToken, startParameter, currency string, prices *[]LabeledPrice, photoUrl string, photoSize, photoWidth, photoHeight int, needName, needPhoneNumber, needEmail, needShippingAddress, isFlexible bool) InvoiceConfig {
```
Comment (Syfaro) [inline helpers.go]: Only required fields should be passed to it, anything not required should just be set on the config afterwards. All the optional fields should just be set to sane defaults.

File: configs.go [hunk 4]
```diff
@@ -899,3 +899,87 @@ type ChatConfigWithUser struct {
...
+type InvoiceConfig struct {
+	BaseChat
+	Title               string          // required
+	Description         string          // required
+	Payload             string          // required
+	ProviderToken       string          // required
+	StartParameter      string          // required
+	Currency            string          // required
+	Prices              *[]LabeledPrice // required
+	PhotoUrl            string
```
Comment (Syfaro) [inline configs.go]: Should be capitalized as `PhotoURL`

**Comment by MrYadro** [comment]
I pushed new commit with comments and fixes you requested, please take a look.

**Comment by Syfaro** [comment]
Everything looks fantastic, thanks!

**Pull request merged**